pages/common/tpp.md

```diff
@@ -0,0 +1,11 @@
+# tpp
+
+> Command-Line based presentation tool.
+
+- View a presentation:
+
+`tpp {{filename}}`
+
+- Output a presentation:
+
+`tpp -t {{type}} -o {{outputname}} {{filename}}`
```

Comment:
I think the `outputname` is the output filename for the presentation, and `filename` is the input filename. Perhaps tweaking this would help.